pkg/agent/util/conntrack/conntrack.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conntrack
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+
+	"k8s.io/klog/v2"
+	"k8s.io/utils/exec"
+)
+
+// NoConnectionToDelete is the error string returned by conntrack when no matching connections are found
```

Comment:
Better to add "." at end of the sentence.

Reply:
Thanks, will do.